.github/workflows/refapp-3x-patient-search.yml

```
@@ -14,7 +14,15 @@ jobs:
       run:
         working-directory: ./qaframework-bdd-tests
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      - name: Run db and web containers
+        run: |
+          docker-compose -f docker/docker-compose-refqa-3x.yml up -d
+      - name: Wait for Openmrs instance to start
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost/openmrs/spa/login)" != "200" ]]; do echo "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost/openmrs/spa/login)"; sleep 10; done
```

No need to echo the status code.
```suggestion
      - name: Wait for Openmrs instance to start
        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost/openmrs/spa/login)" != "200" ]]; sleep 10; done
```

---

Also, we can reduce the sleep time to 2 - 3 seconds.